File: 2737-row-with-maximum-ones/row-with-maximum-ones.cpp
class Solution {
public:
    vector<int> rowAndMaximumOnes(vector<vector<int>>& arr) {
        // code here
	    int maxOne=0;
	    int row=0;
	    for(int i=0;i<arr.size();i++)
	    {
            sort(arr[i].begin(),arr[i].end());
	        int one=arr[i].end()-upper_bound(arr[i].begin(),arr[i].end(),0);
	        if(one>maxOne)
	        {
	            row=i;
	            maxOne=one;
	        }
	    }
	    return {row,maxOne};
    }
};

File: 2737-row-with-maximum-ones/row-with-maximum-ones.java
class Solution {
    public int[] rowAndMaximumOnes(int[][] mat) {
        int count=0;
        int maxCount=-1;
        int row=-1;
        for(int i=0;i<mat.length;i++)
        {
            count=0;
            for(int j=0;j<mat[0].length;j++)
            {
                if(mat[i][j]==1)
                count++;
            }
            if(maxCount<count){
                maxCount=count;
                row=i;
            }
        }
        int ans[]={row,maxCount};
        return ans;
    }
}